fe/fe-core/src/main/java/com/starrocks/scheduler/PartitionBasedMvRefreshProcessor.java

```diff
@@ -448,7 +450,8 @@ private RefreshJobStatus doRefreshMaterializedView(TaskRunContext context,
             // ref table of materialized view : refreshed partition names
             refTablePartitionNames = refTableRefreshPartitions.entrySet().stream()
                     .collect(Collectors.toMap(x -> x.getKey().getName(), Map.Entry::getValue));
-            LOG.info("materialized:{}, mvToRefreshedPartitions:{}, refTableRefreshPartitions:{}",
+            LOG.info("Check mv {} to refresh partitions: mvToRefreshedPartitions:{}, " +
```

Comment:
outputing too much logs seems not needed, suggest to output less info logs.

Reply:
Yes. I can do this later.

MV Refresh is not debugged for online cases, logging here is just for better  debugging.